manifests/12_prometheus_rule.yaml

```diff
@@ -8,11 +8,59 @@ metadata:
     role: alert-rules
 spec:
   groups:
-    - name: marketplace.rules
+    - name: marketplace.community_operators.rules
       rules:
-        - alert: community-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="community-operators"}[10m]) > 10
-        - alert: certified-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="certified-operators"}[10m]) > 10
-        - alert: redhat-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="redhat-operators"}[10m]) > 10
+        - record: app_registry:community_operators:1xx_response
+          expr: sum(app_registry_request_total{code=~"1..",opsrc="community-operators"})
+        - record: app_registry:community_operators:2xx_response
+          expr: sum(app_registry_request_total{code=~"2..",opsrc="community-operators"})
+        - record: app_registry:community_operators:3xx_response
+          expr: sum(app_registry_request_total{code=~"3..",opsrc="community-operators"})
+        - record: app_registry:community_operators:4xx_response
+          expr: sum(app_registry_request_total{code=~"4..",opsrc="community-operators"})
+        - record: app_registry:community_operators:5xx_response
+          expr: sum(app_registry_request_total{code=~"5..",opsrc="community-operators"})
+        - alert: community_operators_alert
+          annotations:
+            message: Unable to connect to the default OperatorSource community-operators for {{ $value }}% of requests.
+          expr: (sum(rate(app_registry_request_total{code!~"2..",opsrc="community-operators"}[10m])) / sum(rate(app_registry_request_total{opsrc="community-operators"}[10m])) OR on() vector(0)) > 0.5
+          for: 10m
+          labels:
+            severity: critical
```

Comment:
Severity critical is unlikely to be necessary here. Critical pages people and Wales them up at night, I think warning is sufficient here.

Reply:
@brancz understood - I will update the severity level